deck-of-cards-kata/src/main/java/bnymellon/codekatas/deckofcards/custom/collections/MutableCollection.java

```diff
@@ -21,22 +21,28 @@
 import java.util.function.Predicate;
 import java.util.stream.Stream;
 
-public interface MutableCollection<T> extends RichIterable<T>, Collection<T> {
+public interface MutableCollection<T> extends RichIterable<T>, Collection<T>
+{
 
-    default boolean addAllIterable(Iterable<T> iterable) {
-        if (iterable instanceof Collection<T> collection) {
+    default boolean addAllIterable(Iterable<T> iterable)
+    {
+        if (iterable instanceof Collection<T> collection)
+        {
             return this.addAll(collection);
-        } else {
+        } else
```

Comment:
This is formatted wrong. The else should be on a separate line.

Reply:
fixed